IBPSA/Fluid/Storage/BaseClasses/PartialTwoPortInterface.mo

```diff
@@ -0,0 +1,113 @@
+within IBPSA.Fluid.Storage.BaseClasses;
+partial model PartialTwoPortInterface
+  "Partial model transporting fluid between two ports without storing mass or energy"
```

Comment:
Isn't the description "transporting fluid between two ports without storing mass or energy" out of the scope of this model that does not include any transport or balance equation?
I would rather describe this model as "Partial model with two ports and generic variables".

It seems that `IBPSA.Fluid.Interfaces.PartialTwoPortInterface` has the same issue. In addition, the documentation of `IBPSA.Fluid.Interfaces.PartialTwoPortInterface` includes:

> It is similar to Modelica.Fluid.Interfaces.PartialTwoPortTransport, but it does not include the species balance

which I think is not consistent with the model definition, and should be replaced with:

> It is similar to Modelica.Fluid.Interfaces.PartialTwoPort, but contains additional declarations for physical variables of common use.

Reply:
The description "without storing mass or energy" is to distinguish the model from
```modelica
model TwoPortHeatMassExchanger
  "Partial model transporting one fluid stream with storing mass or energy"
...
```
which instantiates a control volume. So, this base class does not store mass or energy, but a model that extends it may add these equations, as the tank does.

In `PartialTwoPortInterface` I would not add "...does not include the species balance" as `PartialTwoPortTransport` also adds other balances (pressure, mass, trace substances). Also "...variables of common use" is in my opinion quite vague as it is not clear what common use means.

Reply:
My comment is more about "***transporting fluid*** between two ports". I believe this is not a fair description of this model, since we don't have any of the following equations, as in `PartialTwoPortTransport` for example.
```
  // Mass balance (no storage)
  port_a.m_flow + port_b.m_flow = 0;
  // Transport of substances
  port_b.Xi_outflow = inStream(port_a.Xi_outflow);
```
This means that models extending this base class may not transport fluid at all. 
The base class essentially provides two fluid ports and a set of variables. In this respect, it is closer to `PartialTwoPort` than to `TwoPortHeatMassExchanger`, and this "set of variables" makes the difference with `PartialTwoPort` (there is no additional equation).
However, this is not a blocking point and it should not prevent from merging this PR.

Reply:
I see. Let me do a separate PR for clarifying the documentation so we can go ahead and merge this PR.

Reply:
To be addressed through https://github.com/ibpsa/modelica-ibpsa/issues/1796